src/isObject.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * In Javascript, this is a two part check because an Array is technically an Object. isObject() is a shortcut to verify that a data type is an Object Literal.
+ * @example <caption>Verify object contains keys.</caption>
```

Comment:
@tbrandle Update the caption to represent what this function is doing. "Verify object exists and is not an array" or something